reference-implementation/lib/transform-stream.js

```diff
@@ -342,10 +338,11 @@ module.exports = class TransformStream {
     this._writableDone = false;
     this._readableClosed = false;
 
-    this._resolveWrite = undefined;
-
-    this._chunkPending = false;
-    this._chunk = undefined;
+    // readableBackpressure begins in an unknown state, to be determined at
```

Comment:
I find this confusing. Perhaps use an explicit enum like "undefined", "no", "yes" or similar?